proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -118,9 +118,8 @@ message LocalModuleResolveResult {
   buf.alpha.registry.v1alpha1.LocalModulePin pin = 2;
   // The type the reference resolved as.
   ResolvedReferenceType resolved_reference_type = 3;
-  // Indicates whether this reference is resolvable through BSR_HEAD.
-  // False indicates a non-default branch (formerly draft) commit.
-  bool is_bsr_head = 4;
+  reserved 4;
```

Comment:
Every other declaration of `reserved` in this repository save one puts the `reserved` declarations at the top of the message, can you follow this convention?